src/ActionList/Item.tsx

```diff
@@ -32,6 +32,16 @@ export interface ItemProps extends React.ComponentPropsWithoutRef<'div'>, SxProp
    */
   leadingVisual?: React.FunctionComponent<IconProps>
 
+  /**
+   * Icon (or similar) positioned after `Item` text.
+   */
+  trailingIcon?: React.FunctionComponent<IconProps>
```

Comment:
`trailingVisual` to match `leadingVisual`?

Reply:
I like the thought, but these prop names came from design and were specified in https://github.com/primer/components/issues/1069#issuecomment-819017023